examples/gpmdcov/gpmdcov.F90

```diff
@@ -309,13 +309,17 @@ subroutine gpmd_Part
       if(lt%verbose >= 1 .and. myRank == 1) write(*,*) "In prg_get_covgraph_h .."
       mls_ii = mls()
       call prg_get_covgraph_h(sy,nl%nnStructMindist,nl%nnStruct,nl%nrnnstruct,gsp2%nlgcut,graph_h,gsp2%mdim,lt%verbose)
-      if(lt%verbose >= 1 .and. myRank == 1) write(*,*) "Time for prg_get_covgraph_h "//to_string(mls()-mls_ii)//" ms"
+      if(lt%verbose >= 1 .and. myRank == 1) write(*,*) "Time for prg_get_covgraph_h ",(mls()-mls_ii)," ms"
 
 #ifdef DO_MPI
-      do ipt = gpat%localPartMin(myRank), gpat%localPartMax(myRank)
+       !do ipt= gpat%localPartMin(myRank), gpat%localPartMax(myRank)
+       do iptt=1,PartsInRankI(myRank)
+          ipt= reshuffle(iptt,myRank)
```

Comment:
Yes:
  !! PartsInRankI(getNRanks()) stores the number of partitions assigned to rank i
  !! reshuffle(j,i) assigns partition reshuffle(j,i) to rank i for j=1,PartsInRankI(getNRanks())